src/components/Button/Button.module.css

```diff
@@ -0,0 +1,25 @@
+.Button,
+.ButtonActive {
+  font-family: "Varela Round", sans-serif;
+  background-color: rgba(241, 189, 221, 0.363);
+  box-shadow: rgba(117, 68, 98, 0.363) 0px 5px 15px;
+  color: rgba(255, 255, 255);
+  padding: 8px;
+  margin: 3px;
+  border: none;
+  border-radius: 3px;
+  font-size: 14px;
+  cursor: pointer;
+}
```

Comment:
`composes` i neden tercih etmedin? Ya da bu yöntemi yapmaya nasıl karar verdin?

Reply:
Aynı stilleri tekrar yazmamak için birleştirmiştim. Composes'ı bilmiyordum, değiştiriyorum hemen uygun şekilde.